packages/components/pie-webc-testing/src/helpers/set-rtl-direction.ts

```diff
@@ -0,0 +1,7 @@
+import { Page } from '@playwright/test';
+
+export const setRTL = async (page: Page) => {
+    await page.evaluate(() => {
+        document.documentElement.setAttribute('dir', 'rtl');
```

Comment:
I'm sure you've checked this, but just checking that any subsequent tests will be back to LTR by default?

Reply:
It should tear down after the end of the test. I've just tested this and that's what happens so it does go back to LTR.